api/plan.go

```diff
@@ -61,3 +61,12 @@ func removePlan(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 	}
 	return err
 }
+
+func listRouters(w http.ResponseWriter, r *http.Request, t auth.Token) error {
```

Comment:
I think you can keep it there. router.go is completely unrelated in this package. It makes a certain sense to keep it in plan.go since routers are part of plans.